changelog.txt

```diff
@@ -2,6 +2,7 @@ V.Next
 ---------
 - [PATCH] Add JWT header field for KDF version (#2220)
 - [MINOR] Catch unknown Keystore errors during decryption (#2255)
+- [MINOR] Implement updates of the Native Auth API (#2261)
```

Comment:
nit: let's call it "web API", as to avoid confusion between that vs. SDK API.

Reply:
Updated the entry